docs/index.rst

```diff
@@ -0,0 +1,67 @@
+
+.. raw:: html
+
+    <div>
+      <center>
+        <img src="https://img.shields.io/pypi/v/uvloop.svg" />
+        <img src="https://img.shields.io/pypi/status/uvloop.svg?maxAge=2592000?style=plastic" />
+        <img src="https://img.shields.io/travis/MagicStack/uvloop/master.svg" />
+        <img src="https://img.shields.io/pypi/dm/uvloop.svg" />
+        <img src="https://readthedocs.org/projects/uvloop/badge/?version=latest&style=flat" />
+        <br />
+        <br />
+      </center>
+    </div>
+
+uvloop
+======
+
+`uvloop` is a fast, drop-in replacement of the built-in asyncio event loop.
+`uvloop` is released under the MIT license.
+
+`uvloop` and asyncio, combined with the power of async/await in Python 3.5,
+makes it easier than ever to write high-performance networking code in Python.
+
+`uvloop` makes asyncio fast. In fact, it is at least 2x faster than nodejs,
+gevent, as well as any other Python asynchronous framework. The performance of
+uvloop-based asyncio is close to that of Go programs.
```

Comment:
Would be nice to have a link to the [blog post](http://magic.io/blog/uvloop-blazing-fast-python-networking/) here.